server/src/test/java/org/elasticsearch/VersionTests.java

```diff
@@ -101,7 +101,9 @@ public void testMax() {
     }
 
     public void testMinimumIndexCompatibilityVersion() {
+        assertEquals(Version.fromId(6000099), Version.V_7_0_0.minimumIndexCompatibilityVersion());
```

Comment:
I don't think this line is necessary.The 6 -> 5 is sufficient to check the major version minus one logic. Using version constants in these tests is what we want to avoid.